teuthology/task/kernel.py

```diff
@@ -188,10 +188,22 @@ def need_to_install(ctx, role, version):
                                                                  want=version))
 
     if '.' in str(version):
-        # version is utsrelease, yay
         if cur_version == version:
             log.debug('utsrelease strings match, do not need to install')
             ret = False
+        os_type = teuthology.get_distro(ctx)
+        log.debug("Distro of this test job: {}".format(our_os))
+        if os_type in ['sle', 'opensuse']:
+            cur_version_match = re.search('(.*)-default$', cur_version)
+            if cur_version_match:
+                cur_version_rp = cur_version_match.group(1)
+                if cur_version_rp in version:
+                    log.debug('"{}" is a substring of "{}" - the latest {} kernel is running'
+                              .format(cur_version_rp, version, os_type))
```

Comment:
nit: I'm not sure ours should have different format than others, shouldn't it include something like `do not need to install` at the end...

Reply:
No, because when this message is displayed, the kernel package has already been installed. Please see http://qa-proxy.ceph.com/teuthology/smithfarm-2020-06-30_15:09:23-rados:cephadm:smoke-master-distro-basic-smithi/5191850/teuthology.log